src/dotnet/RiderPlugin.UnrealLink/PluginInstaller/UnrealPluginInstaller.cs

```diff
@@ -164,9 +166,15 @@ private void InstallPluginInGame(UnrealPluginInstallInfo unrealPluginInstallInfo
             var range = 1.0 / size;
             for (int i = 0; i < unrealPluginInstallInfo.ProjectPlugins.Count; i++)
             {
+                if (lifetime.IsNotAlive)
```

Comment:
AFAIK there should be helper method to convert lifetime to CancellationToken.
And if it's possible I think it will be better to pass CancellationToken into all InstallPlugin* functions instead of lifetime.
Thus should replace checks like `lifetime.IsNotAlive` to more explicit checks of CancellationToken